src/courtroom.cpp

```diff
@@ -1363,8 +1384,16 @@ void Courtroom::handle_chatmessage(QStringList *p_contents)
       break;
     //case 4 is AO2 only
     case 4:
-      ui_vp_objection->play("custom", f_char, f_custom_theme);
-      objection_player->play("custom.wav", f_char, f_custom_theme);
+      if(objection_custom != "")
+      {
+          ui_vp_objection->play("custom_objections/" + objection_custom, f_char, f_custom_theme);
+          objection_player->play("custom_objections/" + objection_custom.split('.')[0] + ".wav", f_char, f_custom_theme);
```

Comment:
We are supposed to be supporting .ogg and .opus as well.

Reply:
If that's the case I will make so the objection_player searches automatically for all supported formats